Dashboard/Assetfile

```diff
@@ -17,115 +17,6 @@ class LoaderFilter < WebFilters::MinispadeFilter
   end
 end
 
-class EmberAssertFilter < Filter
```

Comment:
You probably need to cleanup `AssetfilePublic` the same way, right?

Reply:
yes youre right. we can do this at a later stage though because the byhbrid build is still getting the job done as it is